src/sentry/api/endpoints/broadcast_details.py

```diff
@@ -80,6 +80,11 @@ def put(self, request: Request, broadcast_id) -> Response:
             update_kwargs["date_expires"] = result["dateExpires"]
         if result.get("cta"):
             update_kwargs["cta"] = result["cta"]
+        if result.get("mediaUrl"):
+            update_kwargs["media_url"] = result["mediaUrl"]
+        if result.get("category"):
+            update_kwargs["category"] = result["category"]
+
```

Comment:
If we would use `source` field on the serializer, there would be no need to create our own `update_kwargs` object

Reply:
Agreed, but let's defer (see above)